stubs/docutils/docutils/frontend.pyi

```diff
@@ -60,6 +61,14 @@ class OptionParser(optparse.OptionParser, SettingsSpec):
     default_error_encoding_error_handler: ClassVar[str]
     config_section: ClassVar[str]
     version_template: ClassVar[str]
+    def __init__(
+        self,
+        components: Tuple[Type[Parser]] = ...,
```

Comment:
As components is sent through `tuple()`, it can be any iterable:

```suggestion
        components: Iterable[Type[Parser]] = ...,
```

(Needs an import of `collections.abc.Iterable`.)

Reply:
Done.